src/main/resources/raster-catalog-default.json

```diff
@@ -82,7 +82,7 @@
     },
     {
       "name":"ibge_bra_biomes",
-      "source_uri":"s3://gfw-data-lake/ibge_bra_biomes/v2004/raster/epsg-4326/{grid_size}/{row_count}/name/geotiff/{tile_id}.tif"
+      "source_uri":"s3://gfw-data-lake/ibge_bra_biomes/v2019/raster/epsg-4326/{grid_size}/{row_count}/name/geotiff/{tile_id}.tif"
```

Comment:
Looks like the tile set had failed due to a typo in the request, it's completing now. Thanks for catching

Reply:
I assume you're also rerunning a rasterization job for landmark, but just as a reminder in case you missed my comment above, s3://gfw-data-lake/landmark_indigenous_and_community_lands/v202407/raster/ still doesn't exist.